components/heap/third_party/dartino/cmpctmalloc.c

```diff
@@ -486,6 +541,38 @@ void cmpct_test_get_back_newly_freed(cmpct_heap_t *heap)
     }
 }
 
+void cmpct_test_churn(cmpct_heap_t *heap)
+{
+    size_t remaining = cmpct_free_size_impl(heap);
+    size_t heap_size = heap->size;
+    const int ITERATIONS = 4096;
+    char *allocations[ITERATIONS];
```

Comment:
Isn't `char *allocations[ITERATIONS] = { NULL };` valid C code that does the same as the loop below?